lib/signaling/v2/peerconnection.js

```diff
@@ -708,8 +712,9 @@ class PeerConnectionV2 extends StateMachine {
           this.emit('connectionStateChanged');
         }
       });
-    } else {
+    } else if (!['disconnected', 'complete'].includes(iceConnectionState)) { // don't stop monitoring for disconnected or completed.
```

Comment:
@makarandp0 I'm not sure other browsers apart from Chrome reach "completed". Can you confirm?

Reply:
interestingly I only saw `completed` on safari. but your comment found an issue: fixed as:  92f0f57